### PR TITLE
refactor(insight): change-point algorithms refactor

### PR DESCRIPTION
**Pull request by Esorakouki**
### PR includes

- add Window-based change point detection
- refactor change-point algorithms
- [ ] fixed #0
- [ ] add / modify test cases
- [ ] documents, demos

### CODE REVIEW

File: packages/lite-insight/src/algorithms/change-point/bayesian.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import BayesianChangePoint, { BreakPoint } from 'bayesian-changepoint';
```
Comment (pddpd) [inline packages/lite-insight/src/algorithms/change-point/bayesian.ts:1]: Should we make words format consistent?

`changepoint`, `breakpoint` or `change point`, `break point`. Here, it seems to be mixed.

File: packages/lite-insight/src/insights/extractors/changePoint.ts [hunk 2]
```diff
@@ -1,36 +1,19 @@
...
 
 export const findChangePoints = (series: number[]): ChangePointItem[] => {
-  const detection = new BayesianChangePoint<number>({
-    breakpointVerifier
-  });
-
-  detection.exec(series);
-
-  const testResult = detection.breakPoints().map(breakPoint => ( { index: breakPoint.index, significance: pettittPValue(series, breakPoint.index) }));
+  const result = ChangePoint.Bayesian(series);
```
Comment (pddpd) [inline packages/lite-insight/src/insights/extractors/changePoint.ts:11]: Why use plural `breakdownValues` in `packages/lite-insight/src/pipeline/insight.ts`, `schemas` in `packages/lite-insight/src/visualization/index.ts`, but singular `result` here.

https://github.com/antvis/AVA/pull/283/files#diff-625a884578cf1bb31d6590d9baec631e24d31a72599da8ccf7348454357ba007R152

https://github.com/antvis/AVA/pull/283/files#diff-e3822dbeeae836666bcecbc5d5769e07c278824b2461612dfa41db84b426b7d5R17

File: packages/lite-insight/src/algorithms/change-point/bayesian.ts (hunk 1 above)
Comment (Esorakouki) [reply packages/lite-insight/src/algorithms/change-point/bayesian.ts:1]: yes, we use `changepoint` in our code and doc

File: packages/lite-insight/src/algorithms/change-point/bayesian.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+type ChangePointItem = {
+  index: number;
+  significance: number;
+};
+
+/**
+ * Bayesian Online Changepoint Detection
+ * https://arxiv.org/pdf/0710.3742.pdf
```
[the remark this answers is not in the document]
Comment (Esorakouki) [reply packages/lite-insight/src/algorithms/change-point/bayesian.ts:22]: deleted. I use an existing package(https://github.com/mathew-kurian/BayesianChangePointJS) directly

**Comment by pddpd** [comment]
LGTM

**Pull request merged**